dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/status/DataSourceStatusChecker.java

```diff
@@ -41,10 +41,18 @@
     @Override
     @SuppressWarnings("unchecked")
     public Status check() {
-        ApplicationContext context = ServiceBean.getSpringContext();
+        ApplicationContext context = null;
+        for (ApplicationContext c : SpringExtensionFactory.getContexts()) {
+            if (c != null) {
+                context = c;
+                break;

```

Comment:
How to make sure this is the right context?

Reply:
I want to  know  how to get  right context , any replace method  ？